code/web/src/modules/user/MyAccount.js

```diff
@@ -0,0 +1,273 @@
+//Imports
+import React, { PureComponent } from 'react'
```

Comment:
Still a bit iffy on PureComponents vs normal components, and I know it vaguely has something to do with a built-in shallow reload, but couldn't speak to it well. I know we're just mirroring their code here but if someone understands it more I'd love to hear an explanation for one over the other.

Reply:
My understanding is that it helps with performance in some cases and doesn't reload as much or often and is best used for small UI POV. [helpful & free article](https://medium.com/groww-engineering/stateless-component-vs-pure-component-d2af88a1200b)